tests/Aspire.EndToEnd.Tests/IntegrationServicesFixture.cs

```diff
@@ -38,7 +38,7 @@ public IntegrationServicesFixture(IMessageSink diagnosticMessageSink)
     {
         _diagnosticMessageSink = diagnosticMessageSink;
         _testOutput = new TestOutputWrapper(messageSink: _diagnosticMessageSink);
-        BuildEnvironment = new(useSystemDotNet: !TestsRunningOutsideOfRepo);
+        BuildEnvironment = new(useSystemDotNet: true);
```

Comment:
Can you help me understand this change? Does this mean that integration tests will no longer require the test sdk ever? Also, does it mean it won't test against real nugets?

Reply:
Yes, it won't require the test sdk ever. Since we don't have a workload anymore, we don't need to install anything into a test sdk, so a system installed sdk can be used. But it will still be running against real nugets when running outside the repo, same as it does now.